apps/client/src/views/cuesheet/cuesheet-table/cuesheet-table-elements/CuesheetHeader.tsx

```diff
@@ -23,8 +23,8 @@ export default function CuesheetHeader(props: CuesheetHeaderProps) {
 
         return (
           <tr key={headerGroup.id}>
-            <th className={style.indexColumn}>{showIndexColumn && '#'}</th>
             <th className={style.actionColumn} />
+            <th className={style.indexColumn}>{showIndexColumn && '#'}</th>
```

Comment:
Is it intended that the column remains when the index is hidden?

Reply:
I was wondering if we should always show the index column. What do you think?

Reply:
well since I was the one who wanted it gone in the first place, I would say no
But if it can be shown in a way that is muted enough, like in the editor where it is clear that is it something other than the cue number